example/cumulus-tf/cumulus_distribution/bucket_map.yaml.tmpl

```diff
@@ -7,12 +7,6 @@ MAP:
 %{ endif ~}
 %{ if length(public_buckets) > 0 ~}
 %{ for bucket in public_buckets ~}
-  cumulus-daac:
-    public-data:
-      browse:
-        bucket: ${bucket}
-        headers:
-          Content-Type: "image/jpeg"
```

Comment:
Removing this since 'public' and 'protected' buckets can only have one path defined in the bucket map.